### ISSUE TITLE
Is this released anywhere?

### ISSUE THREAD

**Issue by i-am-the-slime** [j-mie6/golden-scala#1]
I just checked maven central and couldn't find it but I might be doing something wrong.

**Comment by j-mie6** [comment]
Yes, there is a milestone release made for it; it was very scrappy as it does what it needed to but could do with improvement before an _actual_ release. The coordinates are:  `com.github.j-mie6::golden-scalatest::0.1.0-M1`

**Comment by i-am-the-slime** [comment]
Great! I actually tried this and I'm under the impression that 

```
snapshotsAcceptAll
```

From https://github.com/indoorvivants/snapshot-testing doesn't work with this, is that correct?

**Comment by j-mie6** [comment]
There is no integration between the two, no, but you can accept all changes by using a -dgolden.accept flag (I forget exactly what the syntax is)
